versions.yaml

```diff
@@ -88,8 +88,8 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/qemu/qemu"
```

Comment:
This is the official mirror. The official repo is https://gitlab.com/qemu-project/qemu actually. Just per curiosity (not asking for a change) : why not using the official repo ?

Reply:
So, the whole thing boils down to the time GitLab was not yet being used and QEMU infra would go down and block Kata Containers project.  We can open a different PR for addressing this, if you feel the need.

Reply:
Thanks for the explanation. I think we can live with that.